bootstrap/terraform/vm.tf

```diff
@@ -157,6 +157,7 @@ resource "local_file" "setup_environment_file" {
 export ACCESS_TOKEN_JWT=${data.external.access_token_jwt.result.access_token}
 export VM_NAME=${azurerm_linux_virtual_machine.vault_azure_vm.name}
 export VM_IP_ADDRESS=${azurerm_public_ip.vault_azure_pub_ip.ip_address}
+export RESOURCE_ID=${azurerm_linux_virtual_machine.vault_azure_vm.id}
```

Comment:
I'm guessing this resource ID approach works for VM-based managed identity too?

Reply:
yep!